Evaluator/TA/momentum_evaluator.py

```diff
@@ -48,7 +48,7 @@ def __init__(self):
 
     async def ohlcv_callback(self, exchange: str, exchange_id: str, symbol: str,  time_frame, candle):
         period_length = 14
-        candle_data = self.get_candle_manager(exchange, exchange_id, symbol, time_frame).get_symbol_close_candles().base
+        candle_data = self.get_symbol_candles(exchange, exchange_id, symbol, time_frame).get_symbol_close_candles()
```

Comment:
We still need the ````.base```: get_symbol_close_candles is not returning ```.base```

Reply:
Yes but it is working, I don't really understand why.

The `.base` is returning None and without `.base` a numpy array is returned.